guides/source/5_0_release_notes.md

```diff
@@ -227,6 +227,10 @@ Please refer to the [Changelog][action-pack] for detailed changes.
     `RedirectBackError`.
     ([Pull Request](https://github.com/rails/rails/pull/22506))
 
+*   `ActionDispatch::IntegrationTest` and
+    `ActionController::TestCase` accept keyword arguments instead of
+positional arguments. ([Pull Request](https://github.com/rails/rails/pull/18323))
```

Comment:
`accept` is putting it lightly. Positional args were deprecated in favor of kw args.

Perhaps we shouldn't mention `ActionController::TestCase` because that's soft deprecated.

Another thing to think about: why force people to upgrade a dead end?